sdk/python/tests/integration/feature_repos/repo_configuration.py

```diff
@@ -162,6 +163,7 @@ def construct_test_environment(
     test_repo_config: IntegrationTestRepoConfig,
     test_suite_name: str = "integration_test",
 ) -> Environment:
+    os.environ["IS_TEST"] = "TRUE"
```

Comment:
I think it's a better practice to set global env vars at the CI level, like here https://github.com/feast-dev/feast/blob/master/.github/workflows/pr_integration_tests.yml and here https://github.com/feast-dev/feast/blob/master/.github/workflows/integration_tests.yml, just so that we can have them in one place.

Reply:
Done. Also added it to the Makefile so local tests also include it